Code/GraphMol/DistGeomHelpers/BoundsMatrixBuilder.h

```diff
@@ -38,19 +37,17 @@ RDKIT_DISTGEOMHELPERS_EXPORT void initBoundsMat(DistGeom::BoundsMatPtr mmat,
   This consists of setting 1-2, 1-3 and 1-4 distance based on bond lengths,
   bond angles and torsion angle ranges. Optionally 1-5 bounds can also be set,
   in particular, for path that contain rigid 1-4 paths.
-
   The final step involves setting lower bound to the sum of the vdW radii for
   the remaining atom pairs.
-
   \param mol          The molecule of interest
   \param mmat         Bounds matrix to the bounds are written
   \param set15bounds  If true try to set 1-5 bounds also based on topology
   \param scaleVDW     If true scale the sum of the vdW radii while setting lower
+  \param useMacrocycle14config  If 1-4 distances bound heuristics for macrocycles is used
```

Comment:
This should move down a couple of lines so that it's not in the middle of the description of `scaleVDW`

Reply:
fixed